app/src/sharedTest/java/org/oppia/android/app/databinding/ImageViewBindingAdaptersTest.kt

```diff
@@ -84,7 +84,7 @@ import org.robolectric.annotation.Config
 import org.robolectric.annotation.LooperMode
 import javax.inject.Singleton
 
-/** Tests for [ImageViewBindingAdaptersTest]. */
+/** Tests for [ImageViewBindingAdapters]. */
```

Comment:
I think this shouldn't have square brackets since ImageViewBindingAdapters isn't a class or object.

Reply:
Ah, actually I apologize--I was wrong about this. The square brackets is fine (please actually re-add them). I was conflating this with another case: extension Kotlin files which don't actually have a class or object associated with them.

Reply:
Done!